electron/settings.html

```diff
@@ -51,13 +47,328 @@ <h5 class="nav-group-title">Account Settings</h5>
+              <button id="total-loss-plus-button" class="btn btn-primary counter-adj-button" data-deckid="total" data-winloss="loss" data-direction="up" data-daily="true">+ Loss</button>
+              <button id="total-loss-plus-button" class="btn btn-primary counter-adj-button" data-deckid="total" data-winloss="loss" data-direction="down" data-daily="true">- Loss</button>
+              <button id="daily-total-reset-button" class="btn btn-danger reset-button" data-deckid="total" data-daily="true">Reset</button>
+            </span>
+          </div>
+          <div class="settings-row counter-reset-row" rv-each-deck="dailyCounterDeckList">
+            <span class="counter-reset-span">{deck.name} &ndash; {deck.win} : {deck.loss}</span>
+            <span class="counter-adj-span">
+              <button class="btn btn-primary counter-adj-button" rv-deckid="deck.id" data-winloss="win" data-direction="up" data-daily="true">+ Win</button>
+              <button class="btn btn-primary counter-adj-button" rv-deckid="deck.id" data-winloss="win" data-direction="down" data-daily="true">- Win</button>
+              <button class="btn btn-primary counter-adj-button" rv-deckid="deck.id" data-winloss="loss" data-direction="up" data-daily="true">+ Loss</button>
+              <button class="btn btn-primary counter-adj-button" rv-deckid="deck.id" data-winloss="loss" data-direction="down" data-daily="true">- Loss</button>
+              <button rv-deckid="deck.id" class="btn btn-danger reset-button" data-daily="true">Reset</button>
+            </span>
+          </div>
+        </div>
+      </div>
+      <div id="theme-pane" value="theme" rv-showsettingspane="settingsPaneIndex">
+        <div class="settings-container">
+          <div class="settings-row">
+            <input rv-checked="useFlat" id="enable-flat-theme-toggle" type="checkbox" data-toggle="toggle"> Enable flat theme <span class="helper-text">Changes card display to a simpler "flat" display
+          </div>
+          <div class="settings-row" rv-show="useFlat">
+            <input rv-checked="useMinimal" id="enable-minimal-theme-toggle" type="checkbox" data-toggle="toggle"> Enable minimal theme <span class="helper-text">Streamlines and shrinks the tracker</span>
+          </div>
+          <div class="settings-row">
+            <input rv-checked="showUIButtons" key="showUIButtons" type="checkbox" data-toggle="toggle" class="tf-toggle"> Show the close and minimize buttons
+          </div>
+          <div class="settings-row">
+                <input rv-checked="showHideButton" key="showHideButton" type="checkbox" data-toggle="toggle" class="tf-toggle"> Show the hide tracker button
+          </div>
+          <div class="settings-row">
+                <input rv-checked="showMenu" key="showMenu" type="checkbox" data-toggle="toggle" class="tf-toggle"> Show the menu button
```

Comment:
This toggle should be removed. Consider: user turns on this setting, closes the settings window, then wants to change some other settings. How do they get back?

Reply:
oh, right! forgot about that.

OK, well, my fix will be an additional warning text here then, reminding you that you'll only be able to get to settings from the tray.